.github/actions/publish-docs.sh

```diff
@@ -13,10 +14,16 @@ fi
 
 (
   set -e
+  set -f
```

Comment:
`set -f` turns off pathname expansion in bash, which makes using rsync a lot simpler.